client/src/components/Header/HeaderBgColor.js

```diff
@@ -0,0 +1,22 @@
+// Dictionary of our routes and corresponding background colors. 
+// Accessed by headerBackgroundColor()
+const routeDict = {
```

Comment:
This is good. Keep this but rename this file to match this method and export this one.